learn/cookbooks/docker.md

```diff
@@ -0,0 +1,135 @@
+# Using Meilisearch with Docker
+
+A common way of installing Meilisearch is using Docker. In this guide you will learn what Docker is, how to install it in your local machine, and how to use Meilisearch with it.
+
+## Docker
+
+Docker is a tool that packages an application's code and dependencies into something called a container. Using Docker allows developers to ensure their software runs reliably across a wide range of machines and environments.
+
+Docker containers are distributed in images. Docker images are packages storing not only the container, but also instructions on how Docker should instantiate that container.
+
+## Install Docker
+
+The first step to start using Docker is installing it in your target machine. We recommend following [Docker's official instructions to install Docker Desktop](https://docs.docker.com/get-docker/), available for MacOS, Windows, and Linux.
+
+Docker for Desktop includes an interface for managing your containers, command-line applications and several tools required for using Docker.
```

Comment:
Do you think this is really in the scope of the **meilisearch** documentation?
I don't think I’ve ever seen a software explaining what's docker and how to install it 🤔

Reply:
I was in doubt about these sections. On one hand Docker install instructions are certainly not our business. On the other, I find Docker quite intimidating and confusing for new users, so I thought a little intro might be appreciated by newer users.

Thanks for the feedback. I'll discuss this with rest of team—we will probably reduce it significantly, but might still keep one or two sentences for the sake of inexperienced users.